global_frame: /map
robot_base_frame: /ropod/base_link
update_frequency: 1.0
publish_frequency: 1.0

footprint: [[-0.33, 0.33], [0.33, 0.33], [0.33, -0.33], [-0.33, -0.33]]

# obstacle layer
observation_sources: laser_scan_sensor
laser_scan_sensor:
    sensor_frame: /rslidar_projected_front
    data_type: LaserScan
    topic: /projected_scan
    marking: true
    clearing: true
    min_obstacle_height: 0.0
    max_obstacle_height: 2.0
    obstacle_range: 8.0
    raytrace_range: 10.0
